packages/slice-machine/tests/project/cypress/integration/0-is-running/check.spec.js

```diff
@@ -0,0 +1,7 @@
+/// <reference types="cypress" />
+
+describe("check slicemachine is running", () => {
+  it('visit http://localhost:9999', () => {
+    cy.visit('http://localhost:9999')
```

Comment:
actually found this site that does what we want, this should be cy.visit('/') with the url as environment variable.

Reply:
https://docs.cypress.io/guides/getting-started/testing-your-app#Step-3-Configure-Cypress